src/components/Details/VmDetails/VmDetails.js

```diff
@@ -0,0 +1,158 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { has } from 'lodash';
+import { VmStatus } from '../../VmStatus';
+import {
+  getCpu,
+  getDescription,
+  getFlavor,
+  getMemory,
+  getNodeName,
+  getOperatingSystem,
+  getVmTemplate,
+  getWorkloadProfile,
+} from '../../../utils';
+import { CUSTOM_FLAVOR } from '../../../constants';
+
+const DASHES = '---';
+
+export const getVmIpAddresses = vmi => {
+  let ipAddresses = [];
+  if (has(vmi, 'status.interfaces')) {
+    ipAddresses = vmi.status.interfaces.map(iface => iface.ipAddress);
```

Comment:
I would prefer this as a ternary condition, but there is no real need for this change.